File: 5. Remove spaces from a string/code.cpp
#include <iostream>
#include <string>
using namespace std;

string removeSpaces(string s) {
    string result = "";
    for (char c : s) {
        if (c != ' ') {
            result += c;
        }
    }
    return result;
}

int main() {
    string input_str;
    cout << "Enter a string: ";
    getline(cin, input_str);
    cout << "String after removing spaces: " << removeSpaces(input_str) << endl;
    return 0;
}

File: 5. Remove spaces from a string/code.java
import java.util.Scanner;

public class Main {
    public static String removeSpaces(String s) {
        return s.replaceAll("\\s", "");
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String inputStr = scanner.nextLine();
        System.out.println("String after removing spaces: " + removeSpaces(inputStr));
    }
}
